CHORE: Update installer doc and CI (#329)
ansys/pyaedt-toolkits-antenna: doc/source/getting_started/installation.rst

Installation
============

For users
^^^^^^^^^

> If you are looking for a stable version of the AEDT Antenna Toolkit, we recommend using the

Author: ```suggestion
If you are looking for a stable version of the AEDT Antenna Toolkit, it is recommended to use the
```
